src/sentry/static/sentry/app/views/performance/keyTransactionAction.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {ActionItem} from 'app/views/eventsV2/table/cellAction';
+
+import KeyTransactionQuery from './keyTransactionQuery';
+
+type KeyTransactionActionProps = {
+  projectID: number;
```

Comment:
Is there a preference either way? I've seen both spellings used throughout the code base. Here I choose to use `projectID` because the `KeyTransactionQuery` component uses it and that was simply lifted from the existing `KeyTransactionButton`.

Reply:
We're really inconsistent with `ID` vs `Id`. I have dreams of normalizing it all one day, but it is a lot of code to churn for very little value.

Reply:
I'll update this to `projectId` with the hope that one day it'll be unified.